bootstrapvz/base/manifest-schema.yml

```diff
@@ -6,6 +6,7 @@ required: [name, provider, bootstrapper, system, volume]
 properties:
   name:
     type: string
+  tags: {type: object}
```

Comment:
You should put this in the provider section. We should rarely need to modify the root properties of a manifest. Tags are something provider specific, even though the same concept pops up on most of them, their usage and implementation can be very different.